subtrack.MAUI/Response/SubscriptionsMonthResponse.cs

```diff
@@ -0,0 +1,48 @@
+﻿using subtrack.MAUI.Services;
+using subtrack.DAL.Entities;
+using subtrack.MAUI.Data;
+
+namespace subtrack.MAUI.Response
+{
+    public class SubscriptionsMonthResponse
+    {
+        public DateTime CurrentDate { get; set; }
+        public IEnumerable<Subscription> Subscriptions { get; }
+        public IEnumerable<SubscriptionResponse> SubscriptionResponse { get; }
+
+        public SubscriptionsMonthResponse(DateTime dateTime)
+        {
+            CurrentDate = dateTime;
+        }
+
+        public IEnumerable<SubscriptionResponse> GetSubscriptionCostByMonth()
+        {
+            var subscriptionByMonth = SubscriptionsCalculator.GetSubscriptionListByMonth(Subscriptions, CurrentDate);
+            var subscriptionViewModel = Map(subscriptionByMonth);
+
+            return subscriptionViewModel;
+        }
+
+        public IEnumerable<SubscriptionResponse> Map(IEnumerable<Subscription> Subscriptions)
```

Comment:
Extract the mapping code to its own class/existing mapping class, make use of Linq select statement instead of creating a new list